lib/miq_automation_engine/engine/miq_ae_method_service/miq_ae_service_methods.rb

```diff
@@ -111,7 +111,7 @@ def self.create_automation_request(options, userid = "admin", auto_approve = fal
     end
 
     def self.create_service_provision_request(svc_template, options = nil)
-      result = svc_template.object_send(:provision_request, User.current_user, options)
+      result = svc_template.object_send(:provision_request, User.current_user, options, options[:submit_workflow] => true)
```

Comment:
@d-m-u @eclarizio 
Is the :submit_workflow a new option that was added, and does this need to be fixed even in the REST API?

Reply:
I think the REST API might also need these changes.

https://github.com/ManageIQ/manageiq-api/blob/fa6ab6570125984c47f08990b3ce86877dee46b9/app/controllers/api/base_controller/generic.rb#L209

https://github.com/ManageIQ/manageiq-api/blob/fa6ab6570125984c47f08990b3ce86877dee46b9/app/controllers/api/services_controller.rb#L231

Reply:
As @mkanoor points out this is change is required for more then just automate.  Can we push this change down to a common location like: https://github.com/ManageIQ/manageiq/blob/master/app/models/service_template.rb#L382

Suggesting this work be combined with https://github.com/ManageIQ/manageiq/pull/17839 as they are both trying to solve the same issue.